src/ng5-slider/lib/slider.component.ts

```diff
@@ -1157,22 +1161,28 @@ export class SliderComponent implements OnInit, AfterViewInit, OnDestroy, Contro
 
   // Update position of the floor label
   private updateFloorLab(): void {
-    this.setLabelValue(this.getDisplayValue(this.minValue, LabelType.Floor), this.flrLabElem);
-    this.calculateElementDimension(this.flrLabElem);
-    const position: number = this.viewOptions.rightToLeft
-      ? this.barDimension - this.flrLabElem.dimension
-      : 0;
-    this.setPosition(this.flrLabElem, position);
+    if (!this.flrLabElem.alwaysHide) {
+      this.setLabelValue(this.getDisplayValue(this.minValue, LabelType.Floor), this.flrLabElem);
+      let position: number = 0;
+      if (this.viewOptions.rightToLeft) {
+        this.calculateElementDimension(this.flrLabElem);
+        position = this.barDimension - this.flrLabElem.dimension;
+      }
+      this.setPosition(this.flrLabElem, position);
+    }
   }
 
   // Update position of the ceiling label
   private updateCeilLab(): void {
-    this.setLabelValue(this.getDisplayValue(this.maxValue, LabelType.Ceil), this.ceilLabElem);
-    this.calculateElementDimension(this.ceilLabElem);
-    const position: number = this.viewOptions.rightToLeft
-      ? 0
-      : this.barDimension - this.ceilLabElem.dimension;
-    this.setPosition(this.ceilLabElem, position);
+    if (!this.ceilLabElem.alwaysHide) {
+      this.setLabelValue(this.getDisplayValue(this.maxValue, LabelType.Ceil), this.ceilLabElem);
+      let position: number = 0;
+      if (!this.viewOptions.rightToLeft) {
+        this.calculateElementDimension(this.ceilLabElem);
```

Comment:
Same here, I would call `calculateElementDimension()` regardless of  `rightToLeft`.

Reply:
I updated the pull request to include the requested calls to `this.calculateElementDimension()`